contents/tutorials/android-session-replays.md

```diff
@@ -0,0 +1,267 @@
+---
+title: How to set up Android session replays
+date: 2024-07-10
+author:
+  - lior-neu-ner
+tags:
+  - session replay
+---
+
+[Session replays](/session-replay) are a useful support tool for understanding how users are interacting with your Android app. It also helps you debug and recreate issues. 
+To show how you to set it up with PostHog, in this tutorial we create a basic Kotlin app, add PostHog, and [enable session recordings](/docs/session-replay/mobile#android).
+
+## 1. Create a basic Android app
+
+Our sample app will have two screens:
+
+- The first screen is be a `login` screen with email and password textfields.
+- The second screen is a simple screen with welcome text and logout button.
+
+The first step is to create a new app. Open [Android Studio](https://developer.android.com/studio) and create a new project. Select `No Activity`, name your project `Android-Session-Replays`, and use the defaults for everything else.
```

Comment:
It is fine to have the project be named `Android-Session-Replays` so you don't have to change everything, I just think the title and slug should be singular.